addons/account/static/src/components/account_payment_field/account_payment.xml

```diff
@@ -25,7 +25,7 @@
                                t-on-click.prevent="() => this.assignOutstandingCredit(info.moveId, line.id)">Add</a>
                         </td>
                         <td style="max-width: 11em;">
-                            <a t-att-title="line.date"
+                            <a t-att-title="line.formattedDate"
```

Comment:
Rationale for variable name change in stable?
It's not needed for the bugfix, I think, though it makes up for fw-port conflicts.

Reply:
It is needed. If we overwrite value.date, that one will be parsed again instead of the original one, which is what causes the bug